src/main/java/codeu/model/data/Mention.java

```diff
@@ -0,0 +1,60 @@
+// Copyright 2017 Google Inc.
+
+package codeu.model.data;
+
+import java.time.Instant;
+
+/**
+The Mention class adds the ability to mention a user. 
+It also notifies users who have been mentioned
+ */
+public class Mention {
+
+	  private final UUID idOfMentioner;// the id of this User mentioning another user
+	  private final UUID idOfMentioned; // the id of the user being mentioned by this User
+	  private final String creationOfMessage;  // allows this user to create a message, mentioning another user, using "@"
+	  private final String notificationMessage; // the notification message where this user is being mentioned
+	  private final String[] notificationsOfMentions; // a string Array where each item is a string message-which is a
```

Comment:
I'm not sure I understand what this is?  I don't think this belongs in the base data model.

Reply:
I was thinking to pass the variables: (idOfMentioned, idOfMentioner, creationOfMessage, notificationMessage, notifcationOfMentions) as parameters into the class.

I was thinking the parameters for the class will be assigned  to a user anytime he/she mentions someone else  The id OfMentioned would be the id of the current user mentioning another user.

Should I take a different approach?